app/test_split_integer.py

```diff
@@ -2,20 +2,22 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert (sum(split_integer(6, 2)) == 6), \
+        "sum of parts must be equal to value"
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    assert (split_integer(6, 2) == [3, 3]), "value divisible by parts"
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    assert (split_integer(8, 1) == [8]), "return value if part 1"
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal() -> None:
-    pass
+    assert (split_integer(17, 4) == [4, 4, 4, 5]), "list must be sorted"
 
 
-def test_should_add_zeros_when_value_is_less_than_number_of_parts() -> None:
-    pass
+def test_when_value_is_less_than_number_of_parts() -> None:
+    assert (split_integer(2, 4) == [0, 0, 1, 1]), \
```

Comment:
The parentheses around the assert condition are unnecessary. You can write `assert split_integer(2, 4) == [0, 0, 1, 1]` instead.